about.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="about.css">
+  <title>About | Sheltered animals summit</title>
+</head>
+
+<body>
+  <!-- <div class="header-hero-container"> -->
+  <header>
+    <div class="header-wrapper">
+      <ul class="top-nav-mobile">
+        <li><a href="./about.html" class="item about">About</a></li>
+        <li><a href="#" class="item">Program</a></li>
+        <li><a href="#" class="item">Join</a></li>
+        <li><a href="#" class="item">Sponsor</a></li>
+        <li><a href="#" class="item">News</a></li>
+      </ul>
+      <div class="hamburger">
+        <div class="line"></div>
+        <div class="line"></div>
+        <div class="line"></div>
+      </div>
```

Comment:
- [ ] The only thing missing on your page when you are on the mobile view add how you can go back to the homepage may be add `home` link